Packs/HelloWorld/ReleaseNotes/1_1_12.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### HelloWorld
+The  commands **ip** **domain** and **helloworld-scan-results** were changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
```suggestion
The **ip**, **domain**, and **helloworld-scan-results** commands were changed to return multiple entries (entry per indicator) instead of a single entry.
```